src/History/SearchSection.jsx

```diff
@@ -0,0 +1,18 @@
+export default function SearchSection() {
+  return (
+    <div className="flex justify-between my-[20px] pb-[20px] border-b">
+      <em className="block text-[#d71313] py-[5px]">
+        warning: 최대 50개까지 저장 가능합니다.
+      </em>
+      <input
+        type="text"
+        placeholder="텍스트를 입력해 주세요!"
+        className="w-[45%] border rounded-full text-sm text-[#333] px-[15px]"
+      />
```

Comment:
P2 : warning text와 placeholder는 추후 변경될 수 있는 부분이니까 매개변수로 빼 놓는 것은 어떻까요? 추후에 컴포넌트 구조는 그대로 건들이지 않고 텍스트만 수정하면 깔끔할 것 같아서 의견드립니다
```suggestion
export default function SearchSection({ warningText, placeholder }) {
  return (
    <div className="flex justify-between my-[20px] pb-[20px] border-b">
      <em className="block text-[#d71313] py-[5px]">
        {warningText}
      </em>
      <input
        type="text"
        placeholder={placeholder}
        className="w-[45%] border rounded-full text-sm text-[#333] px-[15px]"
      />
```

Reply:
앗 저희 데이터 저장 최대 용량 수가 지난번 논의했을 때 50개가 최대라고 회의록에 기재되어있어서 고정값인 줄 알았네요^^:; 
상세하게 남겨주신 덕분에 말씀 주신 사항 신속히 수정반영해 뒀습니다!
다만 placeholder같은 경우에는 대부분 텍스트를 입력해주는 고정값이기에 고정으로 두는게 나을 것 같은데 어떻게 생각하시는지 의견 여쭤보아도 될까요?!

Reply:
반영해주셔서 감사합니다:) placeholder 같은 경우에는 반영해주셔도 좋고 넘어가주셔도 좋습니다. 다만 의견을 여쭤주시길래 첨언하자면, placeholder도 결국 사용자에게 표시되는 텍스트이기 때문에 멘트 수정 등으로 유동성이 있을 수도 있겠다는 생각을 했었습니다.